app/controllers/lambda_callback/address_proof_result_controller.rb

```diff
@@ -0,0 +1,23 @@
+module LambdaCallback
+  class AddressProofResultController < AuthTokenController
+    def create
+      dcs = DocumentCaptureSession.new
+      dcs.result_id = result_id_parameter
+      dcs.store_proofing_result(nil, address_result_parameter)
```

Comment:
Do we need to pass the PII back in here? Or can we update this call to not overwrite the PII that's there?

Reply:
I misunderstood and thought some of the PII that was stored was from the result of the resolution proofer, but it doesn't look like that's the case. I'll fix it to store the response without clobbering the stored PII.

Reply:
addressed in https://github.com/18F/identity-idp/pull/4261/commits/2d5518c24dd60ed5750b3ff1bf65e0d25801e14b